fisco-bcos/benchmark/table_benchmark.cpp

```diff
@@ -0,0 +1,252 @@
+    auto context = std::make_shared<ExecutiveContext>();
+    context->setMemoryTableFactory(tableFactory);
+    auto tableFactoryPrecompiled = std::make_shared<precompiled::TableFactoryPrecompiled>();
+    tableFactoryPrecompiled->setMemoryTableFactory(tableFactory);
+    context->setAddress2Precompiled(Address(0x1001), tableFactoryPrecompiled);
+    // BlockInfo blockInfo{h256(), 1, h256()};
+    // context->setBlockInfo(blockInfo);
+    string keyField("key");
+    string valueFields("value,value2");
+    Address origin(1);
+    auto kvTableFactoryPrecompiled = std::make_shared<precompiled::KVTableFactoryPrecompiled>();
+    kvTableFactoryPrecompiled->setMemoryTableFactory(tableFactory);
+
+    auto statetable = tableFactory->openTable(SYS_CURRENT_STATE);
+    auto entry = statetable->newEntry();
+    entry->setField(SYS_VALUE, "0");
+    entry->setField(SYS_KEY, SYS_KEY_CURRENT_NUMBER);
+    statetable->insert(SYS_KEY_CURRENT_NUMBER, entry);
+    tableFactory->commitDB(h256(0), blockNumber++);
+    tableFactory = tableFactoryFactory->newTableFactory(dev::h256(), blockNumber);
+
+
+    auto commitData = [&](int64_t block) {
+        auto statetable = tableFactory->openTable(SYS_CURRENT_STATE);
+        auto entry = statetable->newEntry();
+        entry->setField(SYS_VALUE, to_string(block));
+        entry->setField(SYS_KEY, SYS_KEY_CURRENT_NUMBER);
+        statetable->update(SYS_KEY_CURRENT_NUMBER, entry, statetable->newCondition());
+        tableFactory->commitDB(h256(0), block);
+        tableFactory = tableFactoryFactory->newTableFactory(dev::h256(), block + 1);
+    };
+    auto setPermission = [&](const string& tableName, uint32_t accounts) {
+        auto acTable = tableFactory->openTable(SYS_ACCESS_TABLE);
+        for (uint32_t i = 0; i < accounts; ++i)
+        {
+            auto entry = acTable->newEntry();
+            entry->setField("table_name", tableName);
+            entry->setField("address", Address(i).hex());
+            entry->setField("enable_num", boost::lexical_cast<std::string>(blockNumber));
+            acTable->insert(tableName, entry, std::make_shared<AccessOptions>(origin, false));
+        }
+        commitData(blockNumber++);
+    };
+
+    auto performance = [&](string description, int count, std::function<void()> operation) {
+        auto now = std::chrono::steady_clock::now();
+        operation();
+        std::chrono::duration<double> elapsed = std::chrono::steady_clock::now() - now;
+        cout << "time used(s)=" << std::setiosflags(std::ios::fixed) << std::setprecision(3)
+             << elapsed.count() << " rounds=" << count << " tps=" << count / elapsed.count() << "|"
+             << description << flush;
+        now = std::chrono::steady_clock::now();
+        commitData(blockNumber++);
+        elapsed = std::chrono::steady_clock::now() - now;
+        cout << " | commit time(s)=" << elapsed.count() << endl;
+    };
+
+
+    auto createTable = [&](string prefix, int count) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'prefix' should be passed by const reference.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5208383)